### PR TITLE
feat: add commit-msg

### PR DESCRIPTION
**Pull request by yakhyo**
This is rule requires at least 4 lines of message
Title and body should be seperated by empty line
[#OK] each commet should have tags.

I checked it only with `commit-msg` branch because I could not use `update.sh` because it requires me to be in `main` branch.

### CODE REVIEW

File: .github/hooks/commit-msg [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        BColors[level]
+        + BColors.BOLD
+        + f"{level}: [Policy]"
+        + message
+        + BColors.ENDC
+    )
+
+
+def check_tag(title):
+    _tags = ["chore", "docs", "style", "feat", "fix", "refactor", "test"]
```
Comment (psycoder21) [inline .github/hooks/commit-msg:44]: Update this tags following our commit tag formats.

File: .github/hooks/commit-msg [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    print(
+        BColors[level]
+        + BColors.BOLD
+        + f"{level}: [Policy]"
+        + message
+        + BColors.ENDC
+    )
+
+
+def check_tag(title):
```
Comment (psycoder21) [inline .github/hooks/commit-msg:43]: Please add typing for parameters.

Comment (psycoder21) [reply .github/hooks/commit-msg:43]: Also docstring please

File: update.sh [hunk 3]
```diff
@@ -15,6 +15,10 @@ git pull origin main
 git checkout main
 cd ..
 
+echo "update commit-msg"
+cp .github/hooks/commit-msg .git/hooks/commit-msg
```
Comment (psycoder21) [inline update.sh:19]: this should be `cp .github/hooks/commit-msg ../.git/hooks/commit-msg`

File: update.sh [hunk 4]
```diff
@@ -15,6 +15,10 @@ git pull origin main
...
+chmod +x .git/hooks/commit-msg
```
Comment (psycoder21) [inline update.sh:20]: This should be changed to follow above paths.

**Review by psycoder21 — CHANGES_REQUESTED** [review]
Great work, BTW some changes are required. 😄

**Comment by psycoder21** [comment]
Also you should follow proper PR forms that you can find out at notion page

**Comment by psycoder21** [comment]
Also you shouldn't make PR to main, you have to make PR to develop branch. However, as this repository is not for main development, I'll accept PR to main.

File: update.sh (hunk 3 above)
Comment (yakhyo) [reply update.sh:19]: is it because of symbolic link?

File: .github/hooks/commit-msg (hunk 1 above)
Comment (yakhyo) [reply .github/hooks/commit-msg:44]: Do you mean this tags -:> https://www.notion.so/pyler/Github-ea6117eacf6940d2b89044f78209b44d#f3b30289cf554a84b554af67b4370d3b

File: update.sh (hunk 4 above)
Comment (yakhyo) [reply update.sh]: this is fixed on the last commit

**Review by psycoder21 — APPROVED** [review]
Great work.

**Pull request merged**